File: cses/leetcode/cpp/260.cpp
#include <bits/stdc++.h>
using namespace std;
class Solution
{
public:
    vector<int> singleNumber(vector<int> &nums)
    {
        // get the total of xor bits
        int total = accumulate(nums.begin(), nums.end(),
                               0, bit_xor<int>());
    }
};

File: leetcode/go/260.go
package main

func singleNumber(nums []int) []int {
	mp := make(map[int]int)
	for _, val := range nums {
		mp[val] += 1
	}
	var v []int
	for key, val := range mp {
		if val == 1 {
			v = append(v, key)
		}
	}
	return v
}
